src/components/card/card.jsx

```diff
@@ -8,15 +8,17 @@ import { Category } from "../category";
 
 import * as cardStyles from "./card.module.scss";
 
-const Card = ({ data }) => (
-  <article key={data.id} className={cardStyles.container}>
-    <img className={cardStyles.styledImage} src={data.frontmatter.thumbnail}/>
-    <Category data={data.frontmatter.category}/>
-    <Link className={cardStyles.styledLink} to={`/${_.kebabCase(data.frontmatter.permalink)}`}>
-      {data.frontmatter.title}
-    </Link>
-  </article>
-);
+const Card = ({ data }) => {
```

Comment:
Why this change?

Reply:
I thought that since there is no function or React Hook and also the only thing it returns is html code, in addition to a component, it was correct to put the parenthesis. But I can change it.

Reply:
Because I was testing some functions and hooks, but in the end I didn't use them, so I forgot to change the curly braces again.